src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/AssessmentService.kt

```diff
@@ -95,9 +98,18 @@ class AssessmentService(
   }
 
   @Transactional
-  fun transitionAssessment(prisonNumber: String, event: AssessmentLifecycleEvent, agent: Agent?) {
```

Comment:
Removed as the code check was saying there was too many functions on service...
The performTransition will probably have to move out of the entity at some point because it is on the other side of the JPA proxy there for not subject to the transaction.

Reply:
cool, makes sense to me. 

You can reset the baseline file for detekt by doing `./gradlew detektBaseline` but need to balance this with heeding its warnings.

Reply:
yep, I think `transitionAssessment` this is a good thing to do though